source/content/shibboleth-sso.md

```diff
@@ -62,7 +62,7 @@ Commands below require a [nested docroot](/nested-docroot) structure and should
  composer require simplesamlphp/simplesamlphp
  ```
 
-1. Add a symlink from `web/simplesaml` to `vendor/simplesamlphp/simplesamlphp/www`:
+2. Add a symlink from `web/simplesaml` to `vendor/simplesamlphp/simplesamlphp/www`:
```

Comment:
Each number should be 1. Markdown increments, and this way we can add or remove steps easily.